lib/urbanopt/geojson/building.rb

```diff
@@ -53,7 +53,15 @@ def create_building(create_method, model, origin_lat_lon, runner, zoning=false,
         number_of_stories_above_ground = other_building[:properties][:number_of_stories_above_ground]
         number_of_stories_below_ground = other_building[:properties][:number_of_stories_below_ground]
         number_of_residential_units = other_building[:properties][:number_of_residential_units]
-        space_type = other_building[:properties][:building_type]
+        #space_type = other_building[:properties][:building_type]
```

Comment:
@tanushree04 should put a comment in here saying that we expect the model already has space types defined which are blended for each story.  In general it is good to document any required preconditions for a function.  It would also help explain why this code is commented out